istioctl/pkg/validate/validate_test.go

```diff
@@ -207,7 +207,7 @@ spec:
       - destination:
           host: c
           subset: v2
-        weight: 25`
+        weight: 15`
```

Comment:
Did not understand this change?

Reply:
Just to make this vs invalid. without the pr, it is invalid because the host is empty.

Reply:
I see